CheckDrive.Mobile/CheckDrive.Mobile/CheckDrive.Mobile/ViewModels/ProfileViewModel.cs

```diff
@@ -71,6 +78,33 @@ private async Task OnEditProfileAsync()
         {
             await _navigationService.NavigateToAsync(NavigationPageType.EditProfile);
         }
+        private async Task SaveEditsAsync()
+        {
+            var updatedAccount = new AccountDto
+            {
+                Login = Login,
+                FirstName = FullName.Split(' ')[0],
+                LastName = FullName.Split(' ').Length > 1 ? FullName.Split(' ')[1] : "",
+                Passport = Passport,
+                PhoneNumber = PhoneNumber,
+                Email = Email,
+                Address = Address,
+                Birthdate = DateTime.Parse(Birthdate),
+            };
+
+            var success = await _accountService.UpdateAccountAsync(updatedAccount);
```

Comment:
let's rename it to **result** or something else, because _success_ is not clear.